slides/section_3_software_dev_process.md

```diff
@@ -855,20 +855,6 @@ Time: 10min
 
 <!-- #endregion -->
 
-<!-- #region slideshow={"slide_type": "subslide"} editable=true -->
-### Merge the Feature In
-
-Hopefully you have now refactored the feature to conform to our MVC structure, and ran our regression tests to check that the outputs rermain the same.
-
-We can commit this to our branch, and then switch to the `develop` branch and merge it in.
-
-```bash
-$ git switch develop
-$ git merge full-data-analysis
-```
-
-<!-- #endregion -->
-
```

Comment:
As discussed near the end of our minihack, I think this is actually correct according to the current course material. However, the confusion arises from the Code Review section where it mentions raising a PR for `feature-std-dev` (different from `full-data-analysis` branch worked on here in section 3!) into `main` rather than `develop`, which is the branch everything else has been merged into up until now. I *think* it should be fine to change that PR target to be `develop` but that is probably one we will need to try out on the day.

Reply:
PR to make the change of target branch: #464 . I'll revert the deletion of this slide.